src/task.rs

```diff
@@ -1297,6 +1323,9 @@ struct RawMetrics {
     /// A task poll takes longer than this, it is considered a slow poll.
     slow_poll_threshold: Duration,
 
+    /// A scheduling delay longer than this will be considered a long delay
```

Comment:
nit: being pedantic again:

```suggestion
    /// A scheduling delay of at least this long will be considered a long delay
```